test/e2e/nftables_test.go

```diff
@@ -1,20 +1,32 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"os"
 	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"github.com/Masterminds/semver/v3"
 	"golang.org/x/sync/errgroup"
 
+	"github.com/openshift-kni/commatrix/pkg/client"
 	"github.com/openshift-kni/commatrix/pkg/consts"
-
-	"github.com/openshift-kni/commatrix/pkg/types"
+	commatrixType "github.com/openshift-kni/commatrix/pkg/types"
```

Comment:
Can you just use it as type pkg? It's not really the commatrix type, without la labeling the pkg